nupic/data/file_record_stream.py

```diff
@@ -579,31 +595,15 @@ def isCompleted(self):
     """
     # CSV file is always considered completed
     return True
-  
+
 
   def setCompleted(self, completed=True):
     """ Marks the stream completed (True or False)
     """
     # CSV file is always considered completed, nothing to do
     return
-  
-  
-  def getFieldNames(self):
```

Comment:
Removed in favor of the ones in the base class that provide equivalent functionality.